src/storage/childtries.md

```diff
@@ -1,4 +1,109 @@
-# Cheap Inclusion Proofs: Child Tries
-*[`kitchen/modules/child-trie`](https://github.com/substrate-developer-hub/recipes/tree/master/kitchen/modules/child-trie)*
+# Cheap Membership Proofs: Child Tries
```

Comment:
Is this file the only resource for child tries? I am can't really get what this proof stuff is and how it relates to child tries..

Reply:
This is the only reference in the written portion of the recipes. There are two examples in the kitchen.

I'm also not getting the membership proofs.

Reply:
I think I'll get more clarity on how to construct runtime proofs for child tries during the retreat, but it is a pro of this storage format. In theory, inclusion proofs for child tries are small in size so they can be sent to other chains and/or stored in the runtime.

Reply:
The proofs need to be generated client-side. I don't think I'll look into this for now so I renamed it appropriately.